components/pages/bounty/proposal/view.tsx

```diff
@@ -123,6 +131,18 @@ export default function ProposalPageView({
             />
           </div>
         </div>
+        <div className="row mt-3 mb-5 bg-gray-900 border-radius-8 mx-0">
+        <Comments
+            comments={comments}
+            type="proposal"
+            ids={{
+              issueId: +issue?.id,
+              proposalId: proposal?.id,
+            }}
+            currentUser={userData}
```

Comment:
why do you need the whole current user ?

Reply:
to get to know if there is githubLogin or address.

Reply:
Instead of passing 2 parameters, I made the abstraction for the user object